charts/reports/templates/secret.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if .Values.azureStorageAccountName }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: reports
+type: Opaque
+data:
+  azurestorageaccountname: {{ .Values.azureStorageAccountName | b64enc }}
+  azurestorageaccountkey: {{ .Values.azureStorageAccountKey | b64enc }}
```

Comment:
Those two secrets should be documented in the README, mentionned in the default values.yaml, and the .sops should be modified as well